test/gui/tst_sharing/test.feature

```diff
@@ -265,6 +265,105 @@ Feature: Sharing
         And as "Alice" folder "Parent/localFolder" should exist on the server
 
 
+    Scenario: sharee tries to create a file and a folder inside a shared folder without write permission
+        Given user "Alice" has created folder "Parent" on the server
+        And user "Brian" has been created on the server with default attributes and without skeleton files
+        And user "Alice" has shared folder "Parent" on the server with user "Brian" with "read" permissions
+        And user "Brian" has set up a client with default settings
+        When the user waits for folder "Parent" to be synced
+        And user "Brian" creates a file "Parent/localFile.txt" with the following content inside the sync folder
+            """
+            test content
+            """
+        And user "Brian" creates a folder "Parent/localFolder" inside the sync folder
+        And the user waits for file "Parent/localFile.txt" to be synced
+        And the user waits for folder "Parent/localFolder" to be synced
```

Comment:
how does that work, if that folder actually is never synced?

Reply:
Step `the user waits for file "Parent/localFile.txt" to be synced` doesn't really assert whether the file has synced or not.
Working on it